src/internal/log/loggers.go

```diff
@@ -253,6 +253,7 @@ func InitBatchLogger(logFile string) func(err error) {
 			if keepLog {
 				zap.S().Infof("logfile retained at %v", logFile)
 			}
+			time.Sleep(5 * time.Millisecond)
```

Comment:
Did you mean to retain this?

Reply:
Yeah, it seems to suffer the same fate of being "already closed" like the comment below is talking about even in this branch.

Reply:
Ah, there’s a bigger issue then, with `InitBatchLogger` needing to be held open until all logging has completed. That doesn’t need to be resolved here — I’ve opend CORE-2237 to track it.